loadtest/control/utils_test.go

```diff
@@ -55,3 +55,18 @@ func TestEmulateUserTyping(t *testing.T) {
 	require.NotNil(t, res.Err)
 	require.Equal(t, "an error", res.Err.Error())
 }
+
+func TestGenerateRandomSentences(t *testing.T) {
+	randomize := GenerateRandomSentences(8)
+	s := strings.Split(randomize, " ")
+	require.Len(t, s, 8)
+
+	randomize = GenerateRandomSentences(206)
```

Comment:
Does 206 test anything extra than 8 ? If not, we can probably remove this.

Reply:
It was higher number than the word pool. Not so important, removed.